samples/dotnet_core_dynamic_add_module/README.md

```diff
@@ -0,0 +1,67 @@
+.NET Core Managed Dynamic module addition sample (Linux only for now) 
```

Comment:
Isn't the difference between linux and windows are just the json files. What would be missing for Windows?

Reply:
Ignoring Windows for now.